### ISSUE TITLE
How to view thread-local variables/global variables?

### ISSUE THREAD

**Issue by sonictk** [x13pixels/remedybg-issues#217]
Might be a stupid question, but I have some variables declared with thread-local storage (via `__declspec(thread)`) and I saw in the release notes for 0.3.5.0 that there should now be watch support for these. However when I try to watch my variables I can't seem to get them to show up in the Watch window (in WinDbg I can just enter the name of the variable and it binds correctly). Is there some special syntax I need to use in order for this feature to work?

Thanks!

**Comment by x13pixels** [comment]
Tested with latest release (0.3.5.4) and seems to be working okay with MSVC compiler. I'll need to try again with Clang to see if there is something up (possibly related to #213).

**Comment by sonictk** [comment]
I was using MSVC; let me try to make a small repro case and see if I can get you some screenshots when I get some free time.

EDIT: Ok, I found the problem: if you declare any _static_ thread local storage variable (i.e. `static __declspec(thread) int test = 0;` for example) RemedyBG seems unable to resolve the variable `test` when in the debugger, while WinDbg is ok with doing so. If you remove the `static` specifier, RemedyBG is able to resolve the variable again without issues. This _seems_ like a bug?

(From reading around I understand that TLS objects kind of have static storage implied already, but personally I'm unclear on the "correctness" of this and I'd prefer to leave that explicitly in my code unless there's a good reason it's a bad idea. I'd be happy to learn!) 

Tested against RemedyBG 0.3.5.4 and MSVC 2019 (19.29.30038.1).
